### PR TITLE
RichCommand: Install Inf Driver

### PR DESCRIPTION
**Pull request by marcofranzen99**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- Relates #11481 
- Implements RichCommand for InstallInfDriver
- Fixed error if there is a space in the path of the `.inf` file

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Install button is hidden if no item is selected or in recycle bin
   2. For non-valid `.inf` files, an error message is displayed by windows when executing the command

I could not test a successful installation of an Inf Driver. Maybe someone else can test this, although the installation logic has not changed

### CODE REVIEW

File: src/Files.App/Actions/Content/Install/InstallFontAction.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace Files.App.Actions
+namespace Files.App.Actions.Content.Install
```
Comment (yaira2) [inline src/Files.App/Actions/Content/Install/InstallFontAction.cs:11]: I'm pretty sure that the namespaces for the others actions is just `Files.App.Actions`.

File: src/Files.App/Actions/Content/Install/InstallInfDriverAction.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "Install".GetLocalizedResource();
+
+		public RichGlyph Glyph { get; } = new("\uE9F5");
+
+		public bool IsExecutable => context.SelectedItems.Count == 1 &&
+			FileExtensionHelpers.IsInfFile(context.SelectedItems[0].FileExtension) &&
+			context.PageType is not ContentPageTypes.RecycleBin;
```
Comment (yaira2) [inline src/Files.App/Actions/Content/Install/InstallInfDriverAction.cs:23]: Should it be disabled for archives or not necessary?

**Review by yair100 — DISMISSED** [review]
LGTM

**Comment by yaira2** [comment]
@mafra99 can you fix the merge conflicts?

**Comment by marcofranzen99** [comment]
@yair100 merge conflicts are solved & build is successful

**Pull request merged**